warehouse/templates/packaging/detail.html

```diff
@@ -369,7 +386,7 @@ <h2 class="page-title split-layout">
           </div>
         </div>
 
-        {% if files %}
+        {% if files and not project.lifecycle_status == "quarantine-enter" %}
```

Comment:
Hm, I'm 50/50 on hiding downloads tab while in quarantine. I understand it keeps people from downloading potentially bad stuff, but I'm not sure if it is necessary. If someone really wants to navigate to pypi.org, find the project, click the tab, and download files that seems reasonable to me given the warning banner.

Reply:
The reason I removed the tab is to prevent folks getting annoyed that they cannot download the files - they'll get a 404 on the links on that page.

Would you prefer the links to file list to remain visible, the links to produce 404s?

Reply:
Why will the links be 404s? aren't the links direct to files.pythonhosted.org?

Reply:
🤦 they are - in local dev it's all localhost so I was getting 404s, will remove from this specific spot.